blazor/image-editor/open-save.md

```diff
@@ -38,11 +44,184 @@ Note: To load the image in the image editor, the image is placed within the appl
+    {
+        var imageData = await ImageEditor.GetImageDataAsync();
+        blobUrl = await CreateBlobUrl(imageData);
+    }
+
+    private async Task OpenBlob()
+    {
+        if (!string.IsNullOrEmpty(blobUrl))
+        {
+            await ImageEditor.OpenAsync(blobUrl);
+        }
+    }
+
+    private async Task<string> CreateBlobUrl(byte[] imageData)
+    {
+        var base64String = Convert.ToBase64String(imageData);
+        var jsCode = $@"
+            return new Promise((resolve) => {{
+                const byteCharacters = atob('{base64String}');
+                const byteNumbers = new Uint8Array(byteCharacters.length);
+                for (let i = 0; i < byteCharacters.length; i++) {{
+                    byteNumbers[i] = byteCharacters.charCodeAt(i);
+                }}
+                const blob = new Blob([byteNumbers], {{type: 'image/png'}});
+                resolve(URL.createObjectURL(blob));
+            }});
+        ";
+
+        return await JS.InvokeAsync<string>("eval", jsCode);
+    }
+}
+```
+![Blazor Image Editor with Opening an blob image](./images/blazor-image-editor-save-blob.jpeg)
+
+### Open an image from file uploader 
+
+User can easily open images in the Image Editor using a file uploader. This method allows users to upload an image file from their device and load it directly into the editor. Once the image is selected through the file uploader, pass the file to the open method, and the image will be seamlessly loaded into the editor.
+
+```cshtml
+@using Syncfusion.Blazor.Inputs
+@using Syncfusion.Blazor.ImageEditor
+@using System.IO
+
+<div class="e-img-editor-sample">
+    <SfUploader AllowedExtensions=".jpg,.jpeg,.png" AutoUpload="false" @ref="uploader" Selected="@OnImageSelected">
+    </SfUploader>
+    <SfImageEditor @ref="imageEditor" Height="350px"></SfImageEditor>
+</div>
+
+@code {
+    private SfUploader uploader;
+    private SfImageEditor imageEditor;
+
+    private async Task OnImageSelected(UploadChangeEventArgs args)
+    {
+        if (args.Files != null && args.Files.Count > 0)
+        {
+            var file = args.Files[0];
+            if (file != null)
+            {
+                using (var memoryStream = new MemoryStream())
+                {
+                    await file.File.OpenReadStream(long.MaxValue).CopyToAsync(memoryStream);
+                    string base64String = Convert.ToBase64String(memoryStream.ToArray());
+                    string dataUrl = "data:image/png;base64," + base64String;
+                    await imageEditor.OpenAsync(dataUrl);
+                }
+            }
+        }
+    }
+}
+```
+
+### Open and image from File Manager 
+
+User can easily open images in the Image Editor using the File Manager. This method allows you to browse and select an image file directly from the File Manager and load it into the editor. Once the image is selected, pass the file to the open method, and the image will be seamlessly loaded into the editor.
```

Comment:
pass the file to the OpenAsync method - with API reference. Where is sample code?? if sample not available remove this section

Reply:
sample not available so removed